app/src/main/java/com/likeminds/feedsx/posttypes/view/adapter/PostAdapter.kt

```diff
@@ -51,9 +51,10 @@ class PostAdapter constructor(
         fun savePost() {}
         fun likePost() {}
         fun sharePost() {}
-        fun comment() {}
+        fun comment(postData: PostViewData)
```

Comment:
Why we need complete `postData` in comment?

Reply:
It is required to fetch the post on the PostDetailScreen. We can also pass postId only. But for testing purposes I was passing postData (as I wanted to show commentsCount)